packages/analyzer/src/utils/index.js

```diff
@@ -49,19 +49,20 @@ export const safe = (cb, returnType = '') => {
   } catch {
     return returnType;
   }
-} 
+}
 
 export function withErrorHandling(name, cb) {
   try {
     cb()
-  } catch(e) { 
+  } catch(e) {
+    console.log(e)
```

Comment:
forgot a console.log here

Reply:
good eye.

time for eslint, maybe?

Reply:
by the way, this was in there because `withErrorHandling` swallows the stack